Dockerfile

```diff
@@ -4,7 +4,7 @@ RUN yum install -y python-requests && \
     curl https://raw.githubusercontent.com/openstack/tripleo-repos/master/tripleo_repos/main.py | python - -b stein current-tripleo && \
     yum update -y && \
     yum install -y openstack-ironic-inspector crudini psmisc iproute iptables && \
-    yum clean all
+    yum clean all && rm -rf /var/cache/yum/*
```

Comment:
Same question as on ironic patch: RHEL 8 uses dnf, not yum. On my Fedora /var/cache/yum does not exist.